src/components/contact/assets/contact.scss

```diff
@@ -57,4 +58,23 @@ section.contact {
   color: inherit;
   text-decoration: none;
   border-bottom: 1px solid $black;
+  cursor: pointer;
+}
+.slack_org:hover {
+  animation: slack-org-highlight 200ms ease-out forwards;
```

Comment:
You don't need `animation` for this. You can set the hover state and add a `transition` property to the selector.

Reply:
ohh ya totally forgot about transitions. will update ASAP.